blueflood-core/src/main/java/com/rackspacecloud/blueflood/types/Locator.java

```diff
@@ -30,6 +30,8 @@
     private String tenantId = null;
     private String metricName = null;
 
+    private long lastUpdatedTimestamp;
```

Comment:
Indicate in the name of variable or in the comments if this is seconds or miliseconds.